src/validation/ValidationSchema.js

```diff
@@ -0,0 +1,55 @@
+import * as yup from 'yup';
+import AmountType from '../types/AmountType';
+
+/* eslint-disable import/prefer-default-export */
+export const validationSchema = yup.object({
+  name: yup.string().required('Please enter the scholarship name'),
+  deadline: yup
+    .date()
+    .required('Please enter the scholarship deadline')
+    .typeError('Please enter a valid date'),
+  description: yup
+    .string()
+    .required('Please enter the scholarship description'),
+  website: yup
+    .string()
+    .url('Website must be a valid URL')
+    .required('Please enter the scholarship website'),
+  amount: yup.object().shape({
+    type: yup
+      .mixed()
+      .oneOf(Object.values(AmountType), 'Please choose an option above'),
```

Comment:
Does `.required('Please choose an option above')` work as well?

Reply:
Yes! I changed it